engine/chain/internal/arc/policy.go

```diff
@@ -0,0 +1,36 @@
+package arc
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	chainmodels "github.com/bitcoin-sv/spv-wallet/engine/chain/models"
+	"github.com/bitcoin-sv/spv-wallet/engine/spverrors"
+)
+
+// GetPolicy requests ARC server for the policy
+func (s *Service) GetPolicy(ctx context.Context) (*chainmodels.Policy, error) {
+	result := &chainmodels.Policy{}
+	arcErr := &chainmodels.ArcError{}
+	req := s.prepareARCRequest(ctx).
+		SetResult(result).
+		SetError(arcErr)
+
+	response, err := req.Get(fmt.Sprintf("%s/v1/policy", s.arcCfg.URL))
+
+	if err != nil {
+		return nil, s.wrapRequestError(err)
+	}
+
+	switch response.StatusCode() {
+	case http.StatusOK:
+		return result, nil
+	case http.StatusUnauthorized, http.StatusForbidden:
+		return nil, s.wrapARCError(spverrors.ErrARCUnauthorized, arcErr)
+	case http.StatusNotFound:
+		return nil, spverrors.ErrARCUnreachable
+	}
+
+	return nil, s.wrapARCError(spverrors.ErrARCUnsupportedStatusCode, arcErr)
```

Comment:
Why not put it in a default of a switch? Just curious if there is any particular reason for that